core/sail/shacl/src/main/java/org/eclipse/rdf4j/sail/shacl/config/ShaclSailSchema.java

```diff
@@ -62,6 +62,7 @@
 
 	public final static IRI VALIDATION_RESULTS_LIMIT_TOTAL = create("validationResultsLimitTotal");
 	public final static IRI VALIDATION_RESULTS_LIMIT_PER_CONSTRAINT = create("validationResultsLimitPerConstraint");
+	public final static IRI TRANSACTIONAL_VALIDATION_LIMIT = create("transactionalValidationLimit");
```

Comment:
I'm open to name suggestions!

Our SHACL engine will analyse the changes in the transaction during validation, but to do this it also needs to track all those changes. Large changes also require large amounts of RAM to keep those changes in memory. Furthermore for larger changes it may be faster to just ignore the changes and instead revalidate the entire database during commit(). This last aspect kinda depends on how large your underlying database is.

Reply:
Name sounds ok as it is. The concept it embodies probably needs to be explained in more detail in the documentation though.